README.md

```diff
@@ -50,14 +50,36 @@ Join a Special Interest Group ([SIG](https://github.com/kubernetes/community))
 
 If you're interested in being a contributor and want to get involved in developing Kubernetes, get started with with this reading:
 
-*  The community [expectations](docs/devel/community-expectations.md)
+*  The community [expectations](https://github.com/kubernetes/community/blob/master/contributors/devel/community-expectations.md)
 *  The [contributor guidelines](CONTRIBUTING.md)
-*  The [Kubernetes Developer Guide](docs/devel/README.md)
+*  The [Kubernetes Developer Guide](https://github.com/kubernetes/community/tree/master/contributors/devel)
 
 You will then most certainly gain a lot from joining a [SIG](https://github.com/kubernetes/community), attending the regular hangouts as well as the community [meeting](https://github.com/kubernetes/community/blob/master/community/README.md).
 
 If you have an idea for a new feature, see the [Kubernetes Features](https://github.com/kubernetes/features) repository for a list of features that are coming in new releases as well as details on how to propose one.
 
+### Building Kubernetes for the impatient
+
+If you want to build Kubernetes right away there are two options:
+
+* You have a working [Go environment](https://golang.org/doc/install).
+
+```
+$ go get k8s.io/kubernetes
+$ cd $GOPATH/src/k8s.io/kubernetes
+$ make
```

Comment:
I guess we could do `go get -d k8s.io/kubernetes` that should avoid the warning message as it won't try to build.